meteor-app/imports/ui/assessment/assessment-job-card.js

```diff
@@ -118,7 +118,7 @@ class JobCard extends Component {
 JobCard.propTypes = {
   currentJob: PropTypes.shape({
     _id: PropTypes.string,
-    status: PropTypes.string
+    status: PropTypes.number
```

Comment:
I think a job should always have a status value, so I think you should add `.isRequired` to this